src/features/zaaksysteem/types.ts

```diff
@@ -16,16 +16,91 @@ export type ZaakDetails = {
   indienDatum?: Date;
   registratieDatum?: Date;
   self: string;
-  documenten?: Document[];
+  documenten?: ZaakDocument[];
   omschrijving: string;
 };
 
-export interface Document {
+export interface ZaakDocument {
   id: string;
   titel: string;
   bestandsomvang: number;
   creatiedatum: Date;
   vertrouwelijkheidaanduiding: string;
   formaat: string;
-  inhoud: string;
+  downloadUrl: string;
 }
+
+export type ZaakType = {
+  id: string;
+  onderwerp: string;
+  omschrijving: string;
+  doorlooptijd: string;
+  servicenorm: string;
+};
+
+export type OrganisatorischeEenheidType = {
```

Comment:
Wat mij betreft gaan we geen 'Type'-suffix gebruiken als conventie

Reply:
aangepast.